docs/source/delta-batch.md

```diff
@@ -1196,6 +1196,13 @@ You can also set `delta.`-prefixed properties during the first commit to a Delta
 
 See also the [_](/table-properties.md).
 
+## Syncing table schema and properties to the Hive metastore
+
+You can enable asynchronous syncing of table schema and properties to the metastore by setting `spark.databricks.delta.catalog.update.enabled` to `true`. Whenever the Delta client detects that either of these two were changed due to an update, it will sync the changes to the metastore. The config `spark.databricks.delta.catalog.update.threadPoolSize` can be optionally used to control the size of the threadpool that is used for these asynchronous updates.
```

Comment:
In the documentation, we should also talk about how the schema would be available in HMS table properties with an example.

```
If Schema is small, It will be available as following in HMS table properties:
{
  'spark.sql.sources.schema': '{"name":"col1","type":"string","nullable":true, "metadata":{}},{"name":"col2","type":"string","nullable":true,"metadata":{}}'
}

If Schema is large, the schema will be broken down into multiple parts. Appending them together should give the correct schema. E.g.
{
  'spark.sql.sources.schema.numParts': '4',
  'spark.sql.sources.schema.part.1': '{"name":"col1","type":"string","nullable":tr',
  'spark.sql.sources.schema.part.2': 'ue, "metadata":{}},{"name":"co',
  'spark.sql.sources.schema.part.3': 'l2","type":"string","nullable":true,"meta',
  'spark.sql.sources.schema.part.4': 'data":{}}'
 }

```